src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -203,11 +200,11 @@ def get_experiment_results(self, qi_job):
         return experiment_results
 
     @staticmethod
-    def __validate(job):
+    def __validate(job: Qobj) -> None:
         """ Validates the number of shots, classical bits and compiled Qiskit circuits.
 
         Args:
-            job (QObj): The quantum job with the Qiskit algorithm and quantum inspire backend.
+            job (Qobj): The quantum job with the Qiskit algorithm and quantum inspire backend.
```

Comment:
Now that typehints are in the signature, they should not also be in the docstring.

Reply:
Done. Ran in some trouble with recursive imports (QuantumInspireProver -> QuantumInspireBackend -> QuantumInspireProvider) for the type hint in __init__ from QuantumInspireBackend. Solved it in Quantum_Inspire_Provider.py so there's one extra file.